src/filter/time.rs

```diff
@@ -20,7 +20,7 @@ impl TimeFilter {
                     .ok()
                     .or_else(|| {
                         NaiveDate::parse_from_str(s, "%F")
-                            .map(|nd| nd.and_hms(0, 0, 0))
+                            .map(|nd| nd.and_hms_opt(0, 0, 0).unwrap())
                             .ok()
```

Comment:
```suggestion
                            .and_then(|nd| nd.and_hms_opt(0, 0, 0).ok())
```

Reply:
It shouldn't ever fail with those values, but sure.

Reply:
Oh true, I was looking at `LocalResult` but for `NaiveDate` that should always be valid.